invenio_deposit/config.py

```diff
@@ -161,9 +162,12 @@
     DEPOSIT_RECORDS_UI_ENDPOINTS = {
         '<pid-type>': {
             'pid_type': '<pid-type>',
-            'route': '/path/to/deposit/<pid_value>',
+            'route': '/path/to/deposit/<deposit_type>/<pid_value>',
```

Comment:
`deposit_type` is not a view argument. Hence it would be cleaner to say something like: `/unique/path/to/deposit/<pid_value>`